packages/node/src/index.ts

```diff
@@ -78,7 +78,7 @@ export { defaultIntegrations, init, defaultStackParser, getSentryRelease } from
 export { addRequestDataToEvent, DEFAULT_USER_INCLUDES, extractRequestData } from '@sentry/utils';
 export { deepReadDirSync } from './utils';
 export { getModuleFromFilename } from './module';
-export { enableAnrDetection } from './anr';
+export { enableAnrDetection, isAnrChildProcess } from './anr';
```

Comment:
This smells like merge artifact or something?

Reply:
no, it's needed by the node-experimental client 😬 if we care strongly about not exporting this I can also copy the code over?

Reply:
No it's fine!